projects/@shared/components/properties-item-new.vue

```diff
@@ -196,6 +201,18 @@ export default {
     warnings() {
       return this.validation.warnings.filter(item => (item.path[0] === this.term));
     },
+    valueExceedsLimit() {
+      return this.hasErrors && this.validation.errors
+        .find(item => (item.path[0] === this.term && item?.context?.limit > 0));
+    },
+    splitValue() {
```

Comment:
Persoonlijk vind ik deze naam te generiek. De functie is duidelijk, maar de naam zegt me niet zo veel. Misschien renamen naar: `splitValueByLengthLimit` of iets anders wat daarop lijkt.

Reply:
Changed.